File: PHP PowerPoint Examples/SlidesApiDemo/copySlide.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "SlideCopy1.pptx";
$sourcePassword = null;
$sourceStorage = null;
$password = null;
$folder = "/Slides/SlideCopy/";
$storage = null;
$slideToCopy = 2;
$position = 3;
$source = "Powerpoint/SlideCopy1.pptx";
$apiInstance->copySlide($name, $slideToCopy, $position, $source, $sourcePassword, $sourceStorage, $password, $folder, $storage);

File: Python PowerPoint Examples/SlidesApiDemo/copy_slide/copySlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "SlideCopy.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_to_copy = 2
position = 3
source = "input/SlideCopy.pptx"
result =api.copy_slide(fileName, password=password, storage=storage, folder=inputFolder,slide_to_copy=slide_to_copy, position=position, source=source)